translations/units.tex/pl.po

```diff
@@ -120,6 +120,21 @@ msgid ""
 "\\textbf{Reinforcing} requires that your Town has a Citadel in addition to a Dwelling of that Unit's Level.\\par\n"
 "\n"
 msgstr ""
+"\\hypertarget{Jednostki}{Wraz}\\index{Jednostki} z Bohaterem i powiązaną z nim Talią, gracze posiadają także \\textbf{Talię Kart Jednostek}, która reprezentuje armię poruszającą się z ich Bohaterami.\n"
+"Każda Frakcja ma dostęp do \\textbf{7 różnych Jednostek} o unikalnych zdolnościach i atrybutach.\n"
+"Jednostki są niezbędne do wygrywania Walk i realizacji celów Scenariusza.\n"
+"Każdy Scenariusz posiada instrukcje wskazującą, które Karty Jednostek powinny być zawarte w twojej początkowej Talii Jednostek.\n"
+"Talia ta powinna być oddzielona od reszty Jednostek twojej Frakcji.\\par\n"
+"Każda Jednostka Frakcji jest dwustronna, posiada słabszą stronę \"Garstka\" oraz silniejszą stronę \"Grupa\".\n"
+"Jednostki \"Garstka\" mogą być ulepszone do \"Grupa\" poprzez \\textbf{Wzmocnienie}, które podczas otrzymywania obrażeń mogą zostać zredukowane do strony \"Garstka\".\n"
+"Jednostki powinny być zawsze trzymane po właściwej stronie podczas ich przemieszczania lub sprawdzania.\\par\n"
```

Comment:
a co powiesz na to?

```suggestion
"Karty Jednostek powinny być zawsze odwrócone na właściwą stronę podczas ich przemieszczania lub sprawdzania.\\par\n"
```

Reply:
Zgoda, mówimy tu o Karcie, a nie o Jednostce
Dodane